### ISSUE TITLE
Rollout scorecards across more repos

### ISSUE THREAD

**Issue by planetf1** [open-quantum-safe/tsc#27]
Following the addition of scorecards to [liboqs](https://github.com/open-quantum-safe/liboqs/issues/1706) (pending some final doc updates) we should roll-out to other relevant repositories within oqs.

For oqs - especially as it's the most active repo - we decided to not just add the scorecard generation, but also address the findings.

For rollout, there is ongoing discussion about which repos are production/supported. One option is to at least add scorecard generation to them all. Including publishing. It is just a data point. So merge the capture. Then, how we prioritize any fixes is another matter.

Proposed list (will update based on comments)
- [ ] liboqs (until merged)
- [ ] oqs-provider
- [ ] liboqs-rust
- [ ] ci-containers
- [ ] liboqs-cpp
- [ ] liboqs-go
- [ ] liboqs-python

I've not included docs, demos, dotnet, java, libssh, profiling for now as these are stale or less relevant. demos is worth a discussion ,but as it's an aggregate set of contributions I'd skip for now.

I'm happy to work through these if there's consensus - specifically on the scan/pr/merge. mitigations later.

**Comment by ryjones** [comment]
I can add the bugs for those repos if you like. they will return blank for now.

**Comment by planetf1** [comment]
@ryjones is that change to get the placeholders in the dashboard?

**Comment by ryjones** [comment]
yes. Take a look, I also removed the archived project and split the list up a little.
